theano/sandbox/rng_mrg.py

```diff
@@ -1363,6 +1363,50 @@ def multinomial(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
             raise NotImplementedError(("MRG_RandomStreams.multinomial only"
                                        " implemented for pvals.ndim = 2"))
 
+    def weighted_selection(self, size=None, n=1, pvals=None, ndim=None, dtype='int64',
+                           nstreams=None):
+        """
+        Sample `n` times *WITHOUT replacement* from a multinomial distribution
+        defined by probabilities pvals. `n` needs to be in [1, m], where m is the number of
+        elements to select from, i.e. m == pvals.shape[1]. By default n = 1.
+        
+        Example : WRITEME
+        
+        Notes
+        -----
+        -`size` and `ndim` are only there keep the same signature as other
+        uniform, binomial, normal, etc.
+        TODO : adapt multinomial to take that into account
```

Comment:
@nouiz : I'm guessing that multinomial() was never adapter in order to maintain the interface. Should we take this opportunity to not have these arguments in weighted_selection() or is it more important to maintain consistency with the signatures of uniform, binomial, normal, ...